### ISSUE TITLE
Everything within modalContainer is hidden within React

### ISSUE THREAD

**Issue by kmcaloon** [netlify/netlify-identity-widget#152]
So I am trying to implement the widget within a React Component by initializing then opening the widget within componentDidMount.  The result is that the identity widget iframe displays with it's overlay, but the modalContainer has aria-hidden set to "true", the modalDialog has the class "visuallyHidden", modalContent is missing the email, password, submit fields, etc. Really not sure what I'm doing wrong here and it's a similar setup to the examples I've seen.  Here is the relevant code:

```
import netlifyIdentity from 'netlify-identity-widget';
import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import { isEmpty } from 'lib/utils';
import { withAppState } from 'stores/AppContext';

import './LoginScreen.scss';

//console.log( 'widget styles', widgetStyles );


/**
 * Login Screen.
 */
@withAppState
export default class LoginScreen extends PureComponent {

  /* ------ = REACT = --------------------------------------------------------------------- */
  constructor( props ) {
    super( props );

  }

  /**
   * Component mounted.
   */
  componentDidMount() {

    netlifyIdentity.init({
      container: '#loginscreen'
    });
    netlifyIdentity.open();

  }

  /* ------ = Output = --------------------------------------------------------------------- */
  render() {

    const p = this.props;
    const AppState = p.AppState;

    return(

      <div
      id="loginscreen"
      className={ `LoginScreen ` + ( ! isEmpty( AppState.user ? 'is-showing' : 'is-hidden' ) ) }
      >

      </div>


    )

  }

};



```

**Comment by timmyg** [comment]
@kmcaloon getting the same thing. ever figure it out?

**Comment by sw-yx** [comment]
does this work for you? https://www.netlify.com/blog/2017/10/30/how-to-add-netlify-identity-service-to-react-projects/

**Comment by thm-design** [comment]
Having the same, issue. No solution yet. Haven't even been able to make the widget visible via devtools. Will post if I manage to figure it out.

**Comment by thm-design** [comment]
So for me (same basic use case as OP).

The `.visuallyHidden` class is hiding stuff, seems like that class should probably have been removed when triggering `.open()` And `.modalContent` has opacity 0 etc.

Here is a screen cap: http://recordit.co/SGfH4PJ1wv

**Comment by thm-design** [comment]
Ok another update, it is working for me now. This is what I did:
 - Passed the `APIUrl` key during `.init()`

```
netlifyIdentity.init({
  container: "#auth",
  APIUrl: process.env.REACT_APP_NETLIFY_IDENTITY
});
```

**Comment by kmcaloon** [comment]
So I just had a chance to retest this.  I set up a new app, installed the latest version of the widget, and implemented code similar to what I had on the old app.  Now it seems to be working without having to customize.

@thm-design what happens when you update package and use your original code? Is it working now even without proving the APIURL?

**Comment by sw-yx** [comment]
yea its not at all clear to me if this is still an issue - if it is, please report (or better still, send a PR) as i am not actively thinking about this at all

**Comment by thm-design** [comment]
@kmcaloon well 3 days ago when I posted, was the same day I installed the package in a fresh CRA app. So unless there has been an update since then I think the result will be the same. Maybe it has something to do with configuration on Netlify itself? Unfortunately I have moved on to another solution for now as time was pressing. Will open a different issue if I have a chance to investigate more another time. Thanks for responding though.
